addon/components/nrg-appbar.js

```diff
@@ -24,10 +24,6 @@ export default class NrgAppbarComponent extends Component {
     return null;
   }
 
-  get pageTitle() {
```

Comment:
I would personally vote to move this into the `nrg-application` component and let something override it there, other than relying solely on the application service. Let someone pass in a `pageTitle` component but default to the property on the application service.

Reply:
I'm on the fence about this one.
Even in tests the page-title helper can be used.

This:
![image](https://user-images.githubusercontent.com/2756533/142630011-63c7a0a5-5a8a-4322-b6e2-f1126c9c0280.png)
Becomes this:
![image](https://user-images.githubusercontent.com/2756533/142630050-03a338d5-2c37-4998-b602-b276a480097a.png)

I worry that allowing override from the application component will enable someone to keep using the old way of handling the title by bubbling events down to the application route.

Reply:
I see your point. I just didn't know if there would be a case where a user would want to manage the title themselves rather than having to hand it to the application service each time.

Reply:
YAGNI